stalker_ui/src/theme.scss

```
@@ -0,0 +1,75 @@
+@use '@angular/material' as mat;
+@import './palette.scss';
+
+@include mat.core();
+
+:root {
+  --density-scale: -2;
+  --small-gap-size: calc(16px + var(--density-scale) * 4px);
+  --normal-gap-size: calc(24px + var(--density-scale) * 4px);
+}
+
+$primary: mat.define-palette($primary, 500);
+$accent: mat.define-palette($warn, A200, A100, A400);
```

Ici on load la palette warn dans accent et je pense pas que c'est intended? Avec les couleurs que j'ai suggéré, la façon suivante de faire fonctionne:

```suggestion
$primary: mat.define-palette($primary, 500);
$accent: mat.define-palette($accent, 500, 300, 700);
$warn: mat.define-palette($warn);
```